Sources/MongoSwift/ConnectionString.swift

```diff
@@ -40,15 +40,28 @@ internal class ConnectionString {
             }
         }
 
+        if let heartbeatFrequencyMS = options?.heartbeatFrequencyMS {
+            guard let value = Int32(exactly: heartbeatFrequencyMS), value >= 500 else {
+                throw MongoError.InvalidArgumentError(
+                    message: "heartbeatFrequencyMS must be between 500 and \(Int32.max)"
```

Comment:
can this message include the value they set similar to the maxPoolSize error message below?

Reply:
done